Gopkg.lock

```diff
@@ -1074,6 +1074,27 @@
   pruneopts = "UT"
   revision = "7fb2782df3d83e0036cc89f461ed0422628776f4"
 
+[[projects]]
+  branch = "master"
+  digest = "1:b410da742bf89fa5f255ef1d95bdf42d2a263a0c8f35272b8d5b4faff49017cf"
+  name = "github.com/spotinst/spotinst-sdk-go"
```

Comment:
Can we add a LICENSE to this library please?  I know Apache 2 is OK for kubernetes projects, not sure what other licenses are OK

Reply:
[Added](https://github.com/kubernetes/kops/blob/c70d380492926312495f5b923c5e96e66aa0e843/vendor/github.com/spotinst/spotinst-sdk-go/LICENSE).